json/parse.go

```diff
@@ -132,7 +131,7 @@ func (p *Parser) init(vs structform.Visitor) {
 		currentState: startState,
 	}
 	p.states = p.statesBuf[:0]
-	p.literalBuffer = p.literalBuffer0[:0]
+	p.literalBuffer = make([]byte, 0, 64)
```

Comment:
could you add a comment as to why 64 is the initial capacity?

Reply:
That is hard to tell tbh. The original code used to use `literalBuffer0 [64]byte`.
To my best knowledge so far, there was no issue with this original sizing so I continued to use it. But I can not tell why the original capacity was chosen.
As I want to avoid a comment in the style `there used to be something with this capacity` how should this be documented?